.gitignore

```diff
@@ -1,4 +1,5 @@
 obj/
 bin/
 
-*.o
\ No newline at end of file
+*.o
+.vscode/
```

Comment:
nit: I think there are editor settings that can add newlines on save. Its annoying that git complains about this but what can you do :P

Reply:
Finally found this setting.